applications/sink/redis-sink/pom.xml

```diff
@@ -16,7 +16,6 @@
     </parent>
 
     <properties>
-        <spring-cloud-starters.version>3.0.3</spring-cloud-starters.version>
         <embedded-redis.version>2.0.9</embedded-redis.version>
```

Comment:
There is a newer version for this: https://mvnrepository.com/artifact/com.playtika.testcontainers/embedded-redis.

Reply:
I think we should stick on [2.0.11](https://github.com/Playtika/testcontainers-spring-boot/tree/2.0.11/embedded-redis) since that is the last version before they jump to Redis 6.2 for the docker image and Spring Data Redis is tested against the latest [4.0 release](https://docs.spring.io/spring-data/redis/docs/current/reference/html/#requirements).

I will bump to 2.0.11